### ISSUE TITLE
Ubuntu 17.04 can not open 

### ISSUE THREAD

**Issue by caneraydinbey** [giuspen/cherrytree#265]
![image](https://user-images.githubusercontent.com/17731063/28008772-853199f0-6561-11e7-8b30-6d2e8cd3113c.png)

i quitted after first usage. Then i try to open but always white page comes like in the picture.

**Comment by giuspen** [comment]
Can you run from a terminal? Just type cherrytree and hit enter

**Comment by caneraydinbey** [comment]
Same :(

![image](https://user-images.githubusercontent.com/17731063/28014101-176d44c4-6574-11e7-9864-dbd068f81ad0.png)

@giuspen

**Comment by giuspen** [comment]
Yes but it's what is printed to the terminal I'm interested in, could you copy and paste?

**Comment by caneraydinbey** [comment]
![image](https://user-images.githubusercontent.com/17731063/28022212-2b635aee-6593-11e7-8088-84270e71acdd.png)

nothing written

**Comment by giuspen** [comment]
after running cherrytree you should not see the terminal prompt unless another instance is already running. please run before "killall cherrytree" and then try again "cherrytree".

**Comment by leggewie** [comment]
@caneraydinbey Did that solve your issue?

**Comment by caneraydinbey** [comment]
@leggewie unfortunately no :(

**Comment by giuspen** [comment]
If you run "ps aux | grep cherrytree" you should spot if an unexpected /unwanted cherrytree instance is running and kill it, this seems your case
